scripts/cluster/join.py

```diff
@@ -7,6 +7,7 @@
 import getopt
 import sys
 import time
+from typing import Dict
```

Comment:
This is necessary due to `# type: Dict[str, str]` typing comment. TBH, might make sense to just remove the typing comment instead. I at least don't get much out of seeing that `json()` would return a dict of strings.